recipe/meta.yaml

```diff
@@ -64,6 +65,9 @@ test:
     - gmt-config --all  # [not win]
     - gmt  # [win]
     - gmt grdmath -Rg -I1 0 90 SDIST = dist_to_NP.nc -x2
+    # Test reading/writing a netCDF/HDF5 file
+    - gmt grdcut @earth_relief_01d=gd:netCDF -R5/10/3/5 -Greliefcut.nc
```

Comment:
Won't this use the `libgdal-netcdf` driver instead?

Reply:
Tried the `HDF5` driver in https://github.com/conda-forge/gmt-feedstock/pull/293/commits/f374cf3602282d9c8e0365f9e170eae38767df1b. It also doesn't work https://github.com/conda-forge/gmt-feedstock/runs/27696142205.